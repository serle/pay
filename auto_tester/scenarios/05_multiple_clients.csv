type,client,tx,amount
deposit,1,1,100.0
deposit,2,2,200.0
deposit,3,3,300.0
withdrawal,1,4,25.0
withdrawal,2,5,50.0
dispute,3,3,
