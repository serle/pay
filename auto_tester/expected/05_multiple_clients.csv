client,available,held,total,locked
1,75.0000,0.0000,75.0000,false
2,150.0000,0.0000,150.0000,false
3,0.0000,300.0000,300.0000,false
